core/services/cable_guy/main.py

```diff
@@ -9,11 +11,41 @@
 from fastapi import Body, FastAPI, HTTPException, status
 from fastapi.staticfiles import StaticFiles
 from fastapi_versioning import VersionedFastAPI, version
+from loguru import logger
 from starlette.responses import Response as StarletteResponse
 
 from api import manager
 from api.manager import EthernetInterface
 
+parser = argparse.ArgumentParser(description="Cable Guy service for Blue Robotics Companion")
+parser.add_argument("-v", "--verbose", help="increase output verbosity", action="store_true")
+
+args = parser.parse_args()
+
+if not args.verbose:
+    logger.remove()
+    logger.add(sys.stderr, format="<green>{time:HH:mm:ss.SSS}</green> | <level>{message}</level>")
+
+
+class InterceptHandler(logging.Handler):
+    def emit(self, record) -> None:  # type: ignore
```

Comment:
`from logging import LogRecord`

Reply:
Also, check: loguru_mypy

Reply:
This is basically the same discussion we had in the SITL PR. When introduced, LogRecord raises a lot of other typing problems, that I'm discussing with the Loguru author.

As its replicated code with ArduPilotManager, can we keep it like it is and solve both together on the common library?

Reply:
I got it working here without problems only doing what I said

Reply:
```
core/services/cable_guy/main.py:27: error: Incompatible types in assignment (expression has type "int", variable has type "str")
Found 1 error in 1 file (checked 1 source file)
```

You're sure you're not receiving this as soon as LogRecord is introduced?

Reply:
I just used: level: Union[int, str]

Reply:
Tracked on #313